### DISCUSSION TITLE
set current English Squiggly shortcut

### DISCUSSION THREAD

**Discussion by styw1234** [sumatrapdfreader/sumatrapdf#2913] [Ideas]
How to set current English "Squiggly" shortcut with "w"?

**Comment by GitHubRulesOK** [comment]
type `squi` into https://www.sumatrapdfreader.org/docs/Commands you should see `CmdCreateAnnotSquiggly`

so you need
```
Shortcuts [
	[
		Cmd = CmdCreateAnnotSquiggly
		Key = w
	]
]
```
![image](https://user-images.githubusercontent.com/12754913/187713591-011de9a6-cb61-47ed-b6a9-02e893068829.png)

**Comment by GitHubRulesOK** [comment]
@kjk 
after using above setting I get unexpected crash in pre-release but not 3.4.5
[SumatraPDF-settings.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/9462823/SumatraPDF-settings.zip)
